### PR TITLE
task/update_doc_r5_fiware

### PR DESCRIPTION
**Pull request by frbattid**
Updates the documentation regarding FIWARE Release 5.

I encourage the reviewers to pay special attention to the links in mkdocs.yml :)

### CODE REVIEW

File: doc/manuals/installation_and_administration_manual/gui.md [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+* [Administration](#section5)
+    * [Logging traces](#section5.1)
+    * [Database](#section5.2)
+* [Annexes](#section6)
+    * [Annex A: Creating and installing a RSA identity](#section6.1)
+    * [Annex B: Creating a self-signed certificate](#section6.2)
+    * [Annex C: Binding the GUI to a port under TCP/1024](#section6.3)
+
+##<a name="section1"></a>Introduction
+
```
Comment (pcoello25) [inline doc/manuals/installation_and_administration_manual/gui.md]: White line :)

File: doc/manuals/installation_and_administration_manual/gui.md [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+[Top](#top)
+
+###<a name="section6.3"></a>Annex C: Binding the GUI to a port under TCP/1024
+This GUI may run in any port the user configures. Nevertheless, most of the Unix-like systems avoid non sudoer users to bind applications to ports under 1024. Since the `cosmos-gui` user is not a sudoer one, you won't be able to run the GUI in the typical TCP/443 port, for instance.
+
+In order to solve this, there are several possibilities. One of them is setting the `cap_net_bind_service` <i>capability</i>:
+
+    $ setcap cap_net_bind_service=+ep /path/to/cosmos-gui
+
+Nevertheless, the above shows some caveats, for instance, it is only valid for kernels over 2.6.24, and Linux will disable `LD_LIBRARY_PATH` on any program that has elevated privileges like `setcap` or `suid`.
```
Comment (pcoello25) [inline doc/manuals/installation_and_administration_manual/gui.md]: Does "caveats" exists? :confused:

File: mkdocs.yml [hunk 3]
```diff
@@ -14,29 +14,23 @@ pages:
...
-      - 'Stream processing: Sinfonier Drawer': 'installation_and_administration_manual/streaming/stream_processing_sinfonier_drawer.md'
-      - 'Stream processing: Sinfonier API': 'installation_and_administration_manual/streaming/stream_processing_sinfonier_api.md'
-      - 'Stream processing: Sinfonier BackEnd': 'installation_and_administration_manual/streaming/stream_processing_sinfonier_backend.md'
-      - 'Annex A: Creating a public-private key pair and installing the public key': 'installation_and_administration_manual/batch/annex_a_create_and_install_keypair.md'
-      - 'Annex B: Get a keystone auth token': 'installation_and_administration_manual/batch/annex_b_get_keystone_auth_token.md'
-      - 'Annex C: Deploy Apache Storm Cluster': 'installation_and_administration_manual/streaming/annex_c_deploy_storm_cluster.md'
+      - 'Some words about Cosmos and its ecosystem': 'installation_and_administration_manual/some_words_about_cosmos_and_ecosystem.md'
+      - 'HAAS engine (Shara version)': 'installation_and_administration_manual/haas_engine_sahara.md'
```
Comment (pcoello25) [inline mkdocs.yml]: Shara -> Sahara

Comment (frbattid) [inline mkdocs.yml]: Fixed in https://github.com/telefonicaid/fiware-cosmos/pull/214/commits/a37133d55f651e5407b3eb87fb9192bc1a85c89e

File: doc/manuals/installation_and_administration_manual/gui.md (hunk 2 above)
Comment (frbattid) [inline doc/manuals/installation_and_administration_manual/gui.md]: Fixed in https://github.com/telefonicaid/fiware-cosmos/pull/214/commits/a37133d55f651e5407b3eb87fb9192bc1a85c89e

File: doc/manuals/installation_and_administration_manual/gui.md (hunk 1 above)
Comment (frbattid) [inline doc/manuals/installation_and_administration_manual/gui.md]: Fixed in https://github.com/telefonicaid/fiware-cosmos/pull/214/commits/a37133d55f651e5407b3eb87fb9192bc1a85c89e

**Pull request merged**